src/main/scala/com/softwaremill/realworld/db/DbDataSource.scala

```diff
@@ -0,0 +1,18 @@
+package com.softwaremill.realworld.db
+
+import com.zaxxer.hikari.{HikariConfig, HikariDataSource}
+import zio.ZLayer
+
+import java.sql.Connection
+import javax.sql.DataSource
+import scala.util.Try
+
+object DbDataSource:
```

Comment:
hm it's not quite clear what's the role of all these Db classes - do we need all of them? Both `DbContext` and `DbDataSource`?

Reply:
`DbDataSource` provides actual `DataSource` that is used for `DbMigrator` an by endpoints for executing quill effects.
`DbContext` is for quill, it doesn't need actual `DataSource`. 
But probably they don't need to be in separate objects, maybe a single `Db` object with all of them?

Reply:
Yes, and/or some comments :)